### PR TITLE
Adjusts account merklePath API

### PR DESCRIPTION
**Pull request by bkase**
Adds merklePath and leafHash to `account` so you can get the merkle path
of any account (and the extra hash) s.t. you can compute the implied
root from the blockchain state.

### CODE REVIEW

**Comment by bkase** [comment]
I'm still waiting for `make update-graphql` to finish

File: src/lib/mina_graphql/mina_graphql.ml [hunk 1]
```diff
@@ -852,23 +867,19 @@ module Types = struct
           ; zkapp_uri
           } =
         let open Option.Let_syntax in
-        let%bind public_key = public_key in
```
Comment (mrmr1993) [inline src/lib/mina_graphql/mina_graphql.ml:855]: I'm intrigued; I would have assumed this would fail to compile.

**Review by mrmr1993 — APPROVED** [review]

**Review by MartinMinkov — APPROVED** [review]

File: src/lib/mina_graphql/mina_graphql.ml (hunk 1 above)
Comment (bkase) [reply src/lib/mina_graphql/mina_graphql.ml:855]: nope -- the partial account actually had this be non-optional

Comment (mrmr1993) [reply src/lib/mina_graphql/mina_graphql.ml:855]: That's what I thought, I meant I was surprised it worked with these lines gone!

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**